leakcanary-object-watcher-android/src/main/java/leakcanary/ServiceWatcher.kt

```diff
@@ -54,6 +54,10 @@ class ServiceWatcher(private val reachabilityWatcher: ReachabilityWatcher) : Ins
           }
         }
         Handler.Callback { msg ->
+          if (msg.obj !is IBinder) {
```

Comment:
Add comments explaining what this is about, specifically calling out the device and use case

Reply:
Added comment and link to issue.